lib/ansible/template/vars.py

```diff
@@ -105,7 +105,10 @@ def __getitem__(self, varname):
             try:
                 value = self._templar.template(variable)
             except Exception as e:
-                raise type(e)(to_native(variable) + ': ' + e.message)
+                try:
+                    raise type(e)(to_native(variable) + ': ' + e.message)
+                except AttributeError:
+                    raise type(e)(to_native(variable) + ': ' + to_native(e))
```

Comment:
I think it would probably be ok to just consolidate to `to_native(e)` instead of the try/except here.  Thoughts @abadger ?